content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -178,9 +172,7 @@ and [taints and tolerations](/docs/concepts/scheduling-eviction/taint-and-tolera
 
 The following plugins support `WaitForFirstConsumer` with dynamic provisioning:
 
-- [AWSElasticBlockStore](#aws-ebs)
 - [GCEPersistentDisk](#gce-pd)
```

Comment:
Remove

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27